### PR TITLE
Bugfix/water 3363

### PR DESCRIPTION
**Pull request by Stephan-Hough**

### CODE REVIEW

File: src/internal/modules/charge-information/controllers/charge-information-workflow.js [hunk 1]
```diff
@@ -4,26 +4,21 @@ const { hasScope } = require('internal/lib/permissions');
...
+  const toSetUp = request.pre.chargeInformationWorkflows;
+  const review = request.pre.chargeInformationWorkflowsReview;
+  const changeRequest = request.pre.chargeInformationWorkflowsChangeRequest;
+
   const view = {
     back: '/manage',
     ...request.view,
     pageTitle: 'Charge information workflow',
-    licences: { changeRequest, toSetUp, review: sortBy(review, ['chargeVersion.dateRange.startDate']) },
+    licences: { changeRequest, toSetUp, review: { data: [...sortBy(review.data, ['chargeVersion.dateRange.startDate'])], pagination: review.pagination } },
```
Comment (timtamimi) [inline src/internal/modules/charge-information/controllers/charge-information-workflow.js:17]: Not important: Please consider breaking up the stuff on line 17 into multiple lines to make it more readable.

File: src/internal/views/nunjucks/charge-information/workflow-tabs/to-set-up.njk [hunk 2]
```diff
@@ -20,7 +20,7 @@
                 </tr>
             </thead>
             <tbody>
-                {% for row in licences.toSetUp %}
+                {% for row in licences.toSetUp.data %}
                     <tr class="govuk-table__row">
                         <td class="govuk-table__cell">{{row.licence.licenceNumber}}</td>
```
Comment (timtamimi) [inline src/internal/views/nunjucks/charge-information/workflow-tabs/to-set-up.njk:25]: Please consider wrapping `<td class="govuk-table__cell">{{row.licence.licenceNumber}}</td>` with a href to the licence ID

File: src/internal/views/nunjucks/charge-information/workflow-tabs/review.njk [hunk 3]
```diff
@@ -22,7 +22,7 @@
                 </tr>
             </thead>
             <tbody>
-                {% for chargeInformationRow in licences.review %}
+                {% for chargeInformationRow in licences.review.data %}
                     <tr class="govuk-table__row">
                         <td class="govuk-table__cell">{{chargeInformationRow.licence.licenceNumber}}</td>
```
Comment (timtamimi) [inline src/internal/views/nunjucks/charge-information/workflow-tabs/review.njk:27]: Please consider wrapping `<td class="govuk-table__cell">{{row.licence.licenceNumber}}</td>` with a href to the licence ID

File: src/internal/views/nunjucks/charge-information/workflow-tabs/change-request.njk [hunk 4]
```diff
@@ -22,7 +22,7 @@
                 </tr>
             </thead>
             <tbody>
-                {% for chargeInformationRow in licences.changeRequest %}
+                {% for chargeInformationRow in licences.changeRequest.data %}
                     <tr class="govuk-table__row">
                         <td class="govuk-table__cell">{{chargeInformationRow.licence.licenceNumber}}</td>
```
Comment (timtamimi) [inline src/internal/views/nunjucks/charge-information/workflow-tabs/change-request.njk:27]: Please consider wrapping `<td class="govuk-table__cell">{{row.licence.licenceNumber}}</td>` with a href to the licence ID

File: test/internal/modules/charge-information/controllers/charge-information-workflow.js [hunk 5]
```diff
@@ -27,7 +27,9 @@ const createRequest = () => ({
   },
   query: {},
   pre: {
-    chargeInformationWorkflows: [],
+    chargeInformationWorkflows: { data: [{ status: 'to_setup' }, { status: 'to_setup' }, { status: 'to_setup' }], pagination: { totalRows: 1 } },
```
Comment (timtamimi) [inline test/internal/modules/charge-information/controllers/charge-information-workflow.js:30]: Please consider breaking up the stuff here on lines 30,31,32 into multiple lines to make the object more readable.

**Review by timtamimi — APPROVED** [review]

File: src/internal/views/nunjucks/charge-information/workflow-tabs/to-set-up.njk (hunk 2 above)
Comment (Stephan-Hough) [reply src/internal/views/nunjucks/charge-information/workflow-tabs/to-set-up.njk:25]: will run this past UCD maybe and add it to another ticket

File: src/internal/views/nunjucks/charge-information/workflow-tabs/review.njk (hunk 3 above)
Comment (Stephan-Hough) [reply src/internal/views/nunjucks/charge-information/workflow-tabs/review.njk:27]: as above

**Pull request merged**